src/chunk_append/chunk_append.c

```diff
@@ -261,11 +262,37 @@ ts_ordered_append_should_optimize(PlannerInfo *root, RelOptInfo *rel, Hypertable
 	 */
 	Assert(root->parse->sortClause != NIL);
 
-	/* we only support direct column references here */
-	if (!IsA(tle->expr, Var))
+	if (IsA(tle->expr, Var))
+	{
+		/* direct column reference */
+		sort_var = castNode(Var, tle->expr);
+	}
+	else if (IsA(tle->expr, FuncExpr) && list_length(root->parse->sortClause) == 1)
```

Comment:
why `list_length(root->parse->sortClause) == 1` here?

Reply:
added an example why this is required